pkgs/development/tools/documentation/doxygen/default.nix

```diff
@@ -1,45 +1,24 @@
-{ stdenv, fetchurl, perl, python, flex, bison, qt4, CoreServices, libiconv }:
+{ stdenv, cmake, fetchurl, perl, python, flex, bison, qt4, CoreServices, libiconv }:
 
 let
-  name = "doxygen-1.8.6";
+  name = "doxygen-1.8.11";
 in
 stdenv.mkDerivation {
   inherit name;
```

Comment:
Please remove the unnecessary `let ...` from above and define the name directly here. It's now common to use recursive attribute sets. If you add `rec` after `mkDerivation` `name` can be referenced within this derivation (like in `src` below).